src/pkg/fault/item.go

```diff
@@ -0,0 +1,79 @@
+package fault
+
+type itemKind string
+
+const (
+	ItemKindFile          itemKind = "file"
+	ItemKindContainer     itemKind = "container"
+	ItemKindResourceOwner itemKind = "resource_owner"
```

Comment:
It's unclear why `Kind` is required within the constant name. 
- `ItemFile`
- `ItemContainer`

flows better or `GroupFile`, `CatFile`

Reply:
Just a reference to its underlying type, and a way to namespace it from other similarly named consts.